### ISSUE TITLE
MixPanel for Mobile

### ISSUE THREAD

**Issue by githubbob42** [githubbob42/mingle2github2#1845]
**[Mingle Card: 2071](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/2071)**
Edit
----

Don't see the newly added events in Mixpanel. Waiting on Maurya.

Narrative
---------

As a …, I can …, so that …

Acceptance Criteria
-------------------

Track

*   Creating a new ticket (event "mixpanel:TicketAddedToJob")
*   Creating a new job (event "mixpanel:JobCreated")
*   Copying a ticket (event "mixpanel:CopyTicket")
    *   This event will be fired **in addition** to the `mixpanel:TicketAddedToJob` when a Ticket is created via the Copy Ticket button.

Related Stories
---------------

Tasks
-----

{{table query: SELECT Number, Name, Owner, 'Task Status' WHERE Type = Task and Story = THIS CARD}}

Defects
-------

{{table query: SELECT Number, Name, Owner, 'Defect Status' WHERE Type = Defect and 'Related Story' = THIS CARD}}

Test Plan
---------

**Comment by githubbob42** [comment]
Anshuman : 05/29/2014 15:05 CDT
Cannot see those events
